packages/core/src/option/OptionList.css

```diff
@@ -1,6 +1,7 @@
 .saltOptionList {
   background: var(--salt-container-primary-background);
   border: var(--salt-size-border) var(--salt-selectable-borderStyle-selected) var(--salt-selectable-borderColor-selected);
+  border-radius: var(--salt-palette-corner-weak, 0);
```

Comment:
Is this in the spec?

Reply:
@bhoppers2008

Reply:
I spoke to @bhoppers2008 previously about this, he said then that list should have border-radius